api/taskcluster/webhook.go

```diff
@@ -167,18 +169,19 @@ func (b branchInfos) GetNames() []string {
 func processTaskclusterBuild(aeAPI shared.AppEngineAPI, taskGroupID, taskID string, sha string, labels ...string) (bool, error) {
 	ctx := aeAPI.Context()
 	log := shared.GetLogger(ctx)
+	rootURL := os.Getenv("TASKCLUSTER_ROOT_URL")
```

Comment:
I'm debating whether we should handle the lack of this env var gracefully, e.g. default to `taskcluster.net`?

Reply:
In general, we've tried to avoid defaulting, as a way of finding places where the value is hard-coded.  Otherwise, we'll all have a bad day when `taskcluster.net` stops resolving and 100 things where we *thought* we were passing a value explicitly break because they were actually using the default.